recipe/meta.yaml

```diff
@@ -47,7 +47,7 @@ requirements:
     - libaio  # [linux]
     - libiconv  # [osx]
     - tiledb 2.18.*
-    - curl
+    - libcurl
```

Comment:
You can remove `libcurl` from `run`. It is automatically added due to its "run exports". Honestly you could probably remove most/all of the `run` requirements. I know that the `openssl` and `tiledb` recipes also use "run exports"